src/main/java/gregtech/common/blocks/BlockOre.java

```diff
@@ -161,14 +161,14 @@ protected ItemStack getSilkTouchDrop(IBlockState state) {
     public void getSubBlocks(@Nonnull CreativeTabs tab, @Nonnull NonNullList<ItemStack> list) {
         if (tab == CreativeTabs.SEARCH) {
             blockState.getValidStates().stream().filter(state -> state.getValue(STONE_TYPE).shouldBeDroppedAsItem).forEach(blockState -> list.add(getItem(blockState)));
-        } else if (tab == GregTechAPI.TAB_GREGTECH_ORES) {
+        } else if (tab == GregTechAPI.TAB_GREGTECH_ORES && getRegistryName() != null && getRegistryName().getPath().endsWith("_0")) {
```

Comment:
What's this for?

Reply:
when a stone type with id higher than 15 it will create a new ore block with a increasing number at the end. This makes sure only the normal stone variant is shown

Reply:
Can do `state.getValue(STONE_TYPE) == StoneTypes.STONE` for better visibility? Also, netherrack, endstone variants should show too. There's also the thing where addon/pack devs may want custom StoneTypes to show up? Perhaps a variable in `StoneType`?

Reply:
Perhaps this should also just use a check for if the stone type should be dropped as an item? That way it shows the Stone, netherrack, and endstone ores, and also takes into account if an addon/pack dev would want the stone type to show up?